NuguAgents/Sources/CapabilityAgents/Text/TextAgentSourceItem.swift

```diff
@@ -37,7 +37,7 @@ struct TextAgentSourceItem: Decodable {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         text = try container.decode(String.self, forKey: .text)
         token = try container.decode(String.self, forKey: .token)
-        playServiceId = try container.decode(String.self, forKey: .playServiceId)
+        playServiceId = try? container.decodeIfPresent(String.self, forKey: .playServiceId)
```

Comment:
nit : decodeIfPresent는 throw를 리턴하지 않아서 try로 하셔도 됩니다!
```suggestion
        playServiceId = try container.decodeIfPresent(String.self, forKey: .playServiceId)
```